### ISSUE TITLE
Feature: Add command for creating a new project

### ISSUE THREAD

**Issue by alistairjevans** [autostep/autostep-cli#3]
Adding a command for creating a new project from a selection of provided templates.

Example usage:

**Create new Project**
Should create project with a single .as file, containing a single feature and scenario.
Creates an empty  autostep.config.json.
```bash
> autostep new project
Blank project created.
```

**Create new Web Project**
The last part is the template name. Creates a project with the AutoStep.Web extension already installed, with an example interactions file and an example tests file.
```bash
> autostep new project web
Blank web project created.
```

**Comment by alistairjevans** [comment]
The command should create a new project in the current directory by default, but could also take a directory argument to specify where to put it.

**Comment by SivaGudivada** [comment]
By directory argument do you mean something like the `-d/--directory` Option that is passed for run and build commands ?

**Comment by SivaGudivada** [comment]
For the `run` and `build` commands, I noticed the use of `ExistingOnly()`  extension method (for the `DirectoryInfo` class from Sys.CmdLine API) that validates the argument and checks for the existence of the directory. 

Do we want to add the directory option for this `new project` command in the same manner ? Or do we want to create the directory specified if it does not already exist ? the dotnet-cli's `new` command, for example, takes a `--name`/`-o` Option and creates the directory if it does not exist. 

Thoughts ?

**Comment by alistairjevans** [comment]
Yes, like the ``-d`` command.

Let's follow the style of dotnet new; it makes more sense to create the directory when creating a new project.

**Comment by alistairjevans** [comment]
Fixed by #4
